src/Raven.Server/Documents/Revisions/RevisionsStorage.cs

```diff
@@ -1706,10 +1743,50 @@ public Task<IOperationResult> EnforceConfigurationAsync(Action<IOperationProgres
             bool includeForceCreated, // include ForceCreated revisions on deletion in case of no revisions configuration (only conflict revisions config is exist).
             HashSet<string> collections, 
             OperationCancelToken token)
+        {
+            var result = new EnforceConfigurationResult();
+            await PerformRevisionsOperationAsync(onProgress, result, 
+                (ids, res, tk) => new EnforceRevisionConfigurationCommand(this, ids, res, includeForceCreated, tk), 
+                collections, token);
+
+            return result;
+        }
+
+        public async Task<IOperationResult> AdoptOrphanedAsync(Action<IOperationProgress> onProgress,
+            OperationCancelToken token)
+        {
+            var result = new AdoptOrphanedRevisionsCommand.AdoptOrphanedResult();
+            await PerformRevisionsOperationAsync(onProgress, result,
+                (ids, res, tk) => new AdoptOrphanedRevisionsCommand(this, ids, result, tk), 
```

Comment:
should this be `AdoptOrphanedRevisionsCommand(this, ids, res, tk)` ? 
and let's rename `tk` -> `token`

Reply:
can't because 'token' is already a local var, I had to write there name that different then 'token'